sdk/billing/Azure.ResourceManager.Billing/src/Custom/BillingReservationData.Serialization.cs

```diff
@@ -0,0 +1,500 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Follow this https://github.com/Azure/autorest.csharp?tab=readme-ov-file#change-the-implementation-of-serializationdeserialization-method-of-a-property to fix corresponding property.

Reply:
is there a reason that we have to change all of them instead of only introducing a hook for that property?

Reply:
fixed